src/ts/ssh-tcp/services/portForwardingService.ts

```diff
@@ -587,9 +602,12 @@ export class PortForwardingService extends SshService {
 				if (localPort !== null) {
 					// The chosen local port may be different from the requested port. Use the
 					// requested port in the response, unless the request was for a random port.
-					const portResponse = new PortForwardSuccessMessage();
-					portResponse.port =
+					const forwardedPort =
 						portForwardRequest.port === 0 ? localPort : portForwardRequest.port;
+					const portResponse = await this.messageFactory.createSuccessMessageAsync(
```

Comment:
For some reason I cannot find similar place in `PortForwardingService.cs` where `CreateSuccessMessageAsync()` is used.

Reply:
Oh good catch, it was supposed to be called at `PortForwardingService.cs:588`.

(I'm not currently using custom success messages in dev tunnels, but I thought supporting that in the factory could be useful later.)

Reply:
Fixed.